src/wp-includes/rest-api/endpoints/class-wp-rest-block-patterns-controller.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Patterns_Controller class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ * @since      6.0.0
+ */
+
+/**
+ * Core class used to access block patterns via the REST API.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Block_Patterns_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructs the controller.
+	 *
+	 * @since 6.0.0
+	 */
+	public function __construct() {
+		$this->namespace = '__experimental';
```

Comment:
@ntsekouras @gziolo can this namespace be changed? If yes, should it be `wp/v2`?

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/2488/commits/ceedc0a33b0b336e9465bf6322f4464c28f622c8.